grails-app/controllers/com/unifina/controller/core/signalpath/CanvasController.groovy

```diff
@@ -47,6 +47,8 @@ class CanvasController {
 		def beginDate = new Date()
 		def endDate = new Date()
 		def currentUser = SecUser.get(springSecurityService.currentUser.id)
+
+		// TODO: what's the case for request.JSON in editor action? GETs don't have bodies. Where can a POST come from? What can it possibly do with the response?
```

Comment:
It's probably some old remnant from AlgoCanvas times, maybe once used via a `POST` request. I don't think it's used anymore. It can be removed.